recipes/simmer/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "simmer" %}
+{% set version = "0.3.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 7b6bb13189ebcec123e290163dce18d98c697c7fa9d8a78112cce2cdea53379e
+
+build:
+  number: 2
+  skip: True  # [py<36]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - astropy =4.0.1
+    - cerberus =1.3.2
+    - matplotlib =3.0.0
+    - nbconvert =5.6.1
+    - nbsphinx =0.5.1
+    - numpy =1.19.1
+    - openpyxl =3.0.4
+    - pandas =1.0
+    - pillow =7.2.0
+    - pip
+    - pygments =2.6.1
+    - python
+    - pyyaml =5.3.1
+    - scikit-image =0.17.2
+    - scipy >=1.5.2
+    - tqdm =4.48.2
+    - xlrd >=1.2.0
+  run:
+    - astropy =4.0.1
```

Comment:
Do we need suck tight pinnings of version numbers? This is going to make it difficult for the dependency solver to work, especially with this many dependencies.

Reply:
Thanks for the review! I tried pinning these more tightly to see if that would help after it failed to solve the first time — I'll try loosening and seeing how that goes.